csharp/src/Ice/IConnector.cs

```diff
@@ -10,6 +10,6 @@ public interface IConnector
         /// <summary>Creates a transceiver. The transceiver may not be fully connected until its
         /// <see cref="ITransceiver.InitializeAsync"/> method is called.</summary>
         /// <return>The transceiver.</return>
-        Connection Connect(string connectionId);
+        Connection Connect(string connectionId, bool secure);
```

Comment:
param are missing in the doc-comments.

Reply:
Actually, the whole doc comment seems bogus. It's talking about creating transceivers, not connections.